xla/service/gpu/matmul_utils.h

```diff
@@ -32,11 +32,20 @@ limitations under the License.
 #include "xla/types.h"
 #include "xla/xla_data.pb.h"
 
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
+#include "rocm/rocm_config.h"
```

Comment:
Why does it have #if GOOGLE_CUDA here?
I think the include is only needed for ROCM?